src/App.jsx

```diff
@@ -14,70 +15,75 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
+  const selectedGoodClass = 'has-background-success-light';
```

Comment:
this variable is redundant. you can use this class name directly at jsx
```suggestion
```

Reply:
It is not. Just because max length of the line would be more than 80 symbols if I put this class name directly at jsx